Source/Microsoft.Teams.Apps.CompanyCommunicator/Repositories/Extensions/NotificationRepositoryExtensions.cs

```diff
@@ -22,7 +22,7 @@ public static class NotificationRepositoryExtensions
         /// <param name="notification">Draft Notification model class instance passed in from Web API.</param>
         /// <param name="userName">Name of the user who is running the application.</param>
         /// <returns>A task that represents the work queued to execute.</returns>
-        public static async Task CreateDraftNotificationAsync(
+        public static async Task<DraftNotification> CreateDraftNotificationAsync(
```

Comment:
Let's have this just return the newly created ID.

Reply:
Addressed.